### DISCUSSION TITLE
Jest testing AE v3 beta has been released, with Jest 29 support

### DISCUSSION THREAD

**Discussion by IlCallo** [quasarframework/quasar#14874] [Announcements]
Hi everyone!

Jest AE v3 **beta** has been released, with Jest 29 support, as well as many other goodies.
Let us know if you hit any issue :grin: 

https://github.com/quasarframework/quasar-testing/releases/tag/unit-jest-v3.0.0-beta.5

**Comment by damms005** [comment]
Please kindly drop a comment here when Vite is supported too. Subscribing to this thread so I can get notified. I hope to be able to provide early feedback whenever it drops.

Thanks for all you do man!

**Reply by IlCallo** [reply]
Hey there!
As stated at the start of Jest AE README, Jest AE won't support Vite due to many limitations and problems with it: https://github.com/quasarframework/quasar-testing/issues/244#issuecomment-1293671738

Please switch to Vitest if you need unit testing for Vite projects, [we have an AE for that](https://github.com/quasarframework/quasar-testing/tree/dev/packages/unit-vitest) too
